content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst

```diff
@@ -2,6 +2,7 @@
 Pass - Fail quality check
 =========================
 
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
```

Comment:
Also, I'll just make a comment here since I feel like I'm not really trying on my reviews if I leave too few comments :sweat_smile: 

This comment is an extreme nitpick, so feel free to ignore! While skimming your doc, I noticed that you didn't use any admonition blocks. I'm a sucker for colors so I wanted to suggest an opportunity to use one on lines 102 - 104, where it says

> For a complete guide on how to fill out quality
alert forms, view the documentation on :ref:`quality alerts
<quality/quality_management/quality-alerts>`.

What do you think about using a seealso admonition block instead (also using the doc tag with a relative tag to link to internal documentation):

```
.. seealso::
   :doc:`Complete guide on quality alerts <../quality_management/quality_alerts>`
```

Reply:
I second this— the blocks are useful for readers in a lot of ways and colors are nice!